src/power_grid_model_io/data_types/tabular_data.py

```diff
@@ -50,7 +52,7 @@ def get_column(self, table_name: str, column_name: str) -> pd.Series:
         """
         Select a column from a table, while applying unit conversions and value substitutions
         """
-        table_data = self._data[table_name]
+        table_data = self[table_name]
```

Comment:
Does this work?

Reply:
Yep. It is a synonym for `self.__getitem__(table_name)`

Reply:
The `__getitem__()` duntion returns `self._data[table_name]`, but may call the lazy_csv_loader first:
```python
def __getitem__(self, table_name: str):
    if callable(self._data[table_name]):
        self._data[table_name] = self._data[table_name]()
    return self._data[table_name]
```